caffe2/operators/elementwise_mul_op.h

```diff
@@ -43,6 +43,219 @@ struct MulFunctor {
       Context* context) const;
 };
 
+namespace {
+
+template <typename TGrad, typename TIn>
```

Comment:
Instead of moving too many things to the header file, can we explicitly instantiate this function?

Reply:
> BTW what's ICPX?

Intel c/c++ compiler, used to run as icc/icpc. Intel compiler NG, now run as icx/icpx.